setup.py

```diff
@@ -14,8 +19,9 @@
     author_email='info@scrapinghub.com',
     url='http://github.com/scrapinghub/python-scrapinghub',
     platforms = ['Any'],
-    py_modules = ['scrapinghub'],
-    install_requires = ['requests'],
+    py_modules = ['scrapinghub', 'hubstorage'],
```

Comment:
Done with versioning: bumpversion looks cleaner than reading from `VERSION` everywhere. Right, let's update version in master later after proper testing, +1. Also fixed imports, feel free to check.